app/assets/javascripts/spreadsheet.js

```diff
@@ -68,4 +43,40 @@ $(document).ready(function() {
 	    $(escape_characters(form_parent_id)).submit();
 	});
     });
+
+    // The bulk upload form can also be reached directly without opening a modal
+    if($("#spreadsheet-upload-container").length) {
+	load_upload_form();
+    }
 });
+
+
+function load_upload_form()
+{
+    // When the user clicks the "MODS bulk loads" button, a lightbox is opened. The event
+    // "loaded.blacklight.ajax-modal" is fired just before this Blacklight lightbox is shown.
+    var input_controls = [$("#filetypes_1"), $("#filetypes_2"), $("#convert_only"), $("#note_text")]
+    var radio_buttons = input_controls.slice(0, 3)
+    
+    // Adjust the width of the lightbox when it hosts a metadata spreadsheet upload form.
+    $("#spreadsheet-upload-container").parent().parent().css("width", "650px");
+    
+    // Width of the lightbox when hosting a metadata bulk upload log
+    $("#spreadsheet-log-container").parent().parent().css("width", "700")
+    
+    // None of the form controls should be functional until a file has been selected
+    $("#spreadsheet_submit").prop("disabled", true);
+    input_controls.map(disable_control)
```

Comment:
`.map` was added in ES5... so as long as we don't need to support > IE9 no worries here. (I don't think we need to worry about it in argo)